dialogflow_task_executive/node_scripts/dialogflow_client.py

```diff
@@ -158,6 +164,40 @@ def text_cb(self, msg):
         self.queue.put(msg)
         rospy.loginfo("Recieved input")
 
+    def text_action_cb(self, goal):
+        rospy.loginfo("Recieved action goal")
+        # initialize
+        r = rospy.Rate(1)
```

Comment:
Do you mean should I set longer sleep duration time?

Reply:
like `r = rospy.Rate(10)`

Reply:
Maybe it had better removing the `r.sleep` part...

Reply:
> like r = rospy.Rate(10)

Yes.

>Maybe it had better removing the r.sleep part...

For the while loop, it is better to add `sleep` part, otherwise the loop goes too fast.